### ISSUE TITLE
besides map labels, allow to add "speech bubbles" / "thought bubbles" (GNA #22779)

### ISSUE THREAD

**Issue by wesnoth-bugs** [wesnoth/wesnoth#1413]
**[Original submission](https://gna.org/bugs/index.php?22779)** by involution on 2014-10-10

This would just be a fun little feature to add. I imagine it might be nice for certain scripted sequences in single player, and I'm sure it would be used in MP.

(Reproduced on N/A)
Release: 1.13
Priority: 5 - Normal
Severity: 1 - Wish

**Comment by cooljeanius** [comment]
does `[floating_text]` count?

**Comment by CelticMinstrel** [comment]
No, I don't think so, as it appears like a damage number on a location, rises up, and then vanishes again. I think the new Lua labelling API should be able to create something that works as a sort of speech bubble – the idea would be to position the label so that it's above a particular unit's head. It wouldn't have the appearance of a speech bubble, though – it would just be a line of text above their head.
